Mobile/stockdogmobile/screens/login.js

```diff
@@ -65,57 +65,40 @@ export default class Login extends Component {
         keyboardShouldPersistTaps="handled"
         enableOnAndroid={true}>
           <LinearGradient
-            colors={['transparent', 'rgb(87, 122, 142)']}
-            style={containers.generalGradient}
-            >
-          <Image source={require('../assets/logo.png')} style={containers.logo}/>
-            <TextInput
-              style={elements.roundedInput}
-              placeholder="email"
-              placeholderTextColor="#aaaaaa"
-              onChangeText={(email) => this.setState({email})}
+            colors={['transparent', colors.lightBackground]}
+            style={containers.generalGradient}>
+            <Image 
+              source={require('../assets/logoCrop.png')} 
```

Comment:
Importing assets at the top of the page instead of inline would help readability

Reply:
Ooh thanks for letting me know! Will do!